CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```

Comment:
This should be simple like this: https://github.com/openebs/libcstor/blob/master/CODE_OF_CONDUCT.md

(The repo's that don't follow this above simple format, needs to be updated).

Reply:
Thanks. Will update that.